src/main/java/peoplesoft/logic/commands/job/JobMarkCommand.java

```diff
@@ -24,11 +24,10 @@
     public static final String COMMAND_WORD = "mark";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Marks the job identified by the index number used in the displayed job list.\n"
-            + "Parameters: JOB_INDEX\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + ": Marks the chosen job as completed.\n"
+            + "Example: " + COMMAND_WORD + " 2";
```

Comment:
As with `JobDeleteCommand`, adding "Format: " would probably be more consistent. If there is any specific reason behind removing this for these two commands in particular, do disregard this though

Reply:
i just wanted it to be a bit shorter, but now since the result display can scroll, i should just add them back for consistency.